### PR TITLE
Add Package: tag

### PR DESCRIPTION
**Pull request by emiura**
Added Package: tag in tests, so it can be parsed on a page like this: https://qam.suse.de/openqa-summary-tests/Maintenance_SLE_15_SP2_Updates.html

Removed xterm from package list on some packages.

Related poo#77002.

### CODE REVIEW

File: tests/kernel/build_git_kernel.pm [hunk 1]
```diff
@@ -7,6 +7,8 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
+# Package: git-core ncurses-devel gcc flex bison libelf-devel libopenssl-devel
+# make dracut
```
Comment (michaelgrifalconi) [inline tests/kernel/build_git_kernel.pm:11]: Is it planned to use multiple lines? I am thinking about human-readable vs machine-readable

Comment (emiura) [reply tests/kernel/build_git_kernel.pm:11]: @martinsmarcelo said the parser script is able to handle this.

**Review by michaelgrifalconi — APPROVED** [review]
LGTM, great work!

**Review by tjyrinki — APPROVED** [review]
The additions and modifications seem good to me, thank you.

**Pull request merged**